### PR TITLE
Translate "guidelimited" and "jgmd" for de

### PR DESCRIPTION
**Pull request by WebSnke**

### CODE REVIEW

File: _data/trans/de.json [hunk 1]
```diff
@@ -22,10 +22,10 @@
     "guideexplanations": "Die Links oben sind farbkodiert, um die Schwierigkeit des Account-Löschens anzuzeigen:",
     "guidehard": "Kann nicht gelöscht werden, ohne den jeweiligen Support zu kontaktieren",
     "guideimpossible": "Kann nicht gelöscht werden",
-    "guidelimited": "Account can only be deleted if you live in a location with privacy rights or are covered by an applicable policy such as the iOS App Store guidelines",
+    "guidelimited": "Account kann nur gelöscht werden, wenn du an einem Ort mit Datenschutzrechten lebst oder von zutreffenden Richtlinien, wie z.B. den iOS App Store Richtlinien, abgedeckt wirst.
```
Comment (tupaschoal) [inline _data/trans/de.json:25]:
```suggestion
    "guidelimited": "Account kann nur gelöscht werden, wenn du an einem Ort mit Datenschutzrechten lebst oder von zutreffenden Richtlinien, wie z.B. den iOS App Store Richtlinien, abgedeckt wirst.",
```

Missing end quote and comma.

**Review by tupaschoal — CHANGES_REQUESTED** [review]
I've recently changed `extension_browser`, and  `extensionp1` to `extensionp5` to also mention the Firefox extension. Can you check if the translations there remain valid? You can use the https://github.com/jdm-contrib/jdm/blob/master/TRANSLATION_REFERENCE.md file or my last comment on issue #493 for more details. Thanks!

**Comment by WebSnke** [comment]
> I've recently changed `extension_browser`, and `extensionp1` to `extensionp5` to also mention the Firefox extension. Can you check if the translations there remain valid? You can use the https://github.com/jdm-contrib/jdm/blob/master/TRANSLATION_REFERENCE.md file or my last comment on issue #493 for more details. Thanks!

I am not sure about the `extension_browser` translation, but it seems to be correct to me. A bigger issue whether the project should use honorific language for the translations.

**Comment by WebSnke** [comment]
> > I've recently changed `extension_browser`, and `extensionp1` to `extensionp5` to also mention the Firefox extension. Can you check if the translations there remain valid? You can use the https://github.com/jdm-contrib/jdm/blob/master/TRANSLATION_REFERENCE.md file or my last comment on issue #493 for more details. Thanks!
> 
> I am not sure about the `extension_browser` translation, but it seems to be correct to me. A bigger issue whether the project should use honorific language for the translations.

In Germany (and probably also other languages), you use different pronouns depending on who you are talking to. JDM seems to use the honorific ones which are not that common on websites anymore today.

**Comment by WebSnke** [comment]
[fr.json seems to use honorific pronouns too.](https://github.com/jdm-contrib/jdm/blob/380af6209f9d7a0a599c637b2b66142d8be6da0b/_data/trans/fr.json#L14)

**Comment by tupaschoal** [comment]
If it feels too formal and unnatural, feel free to change, it makes sense to me to not use them as well.

**Comment by WebSnke** [comment]
Did you use Goggle Translate to translate the website?

**Comment by tupaschoal** [comment]
> Did you use Goggle Translate to translate the website?

I did not, we relied on volunteer translations, but then, if it's a language I'm unfamiliar with, I couldn't really tell if they themselves used Google Translator.

**Comment by tupaschoal** [comment]
(Btw, let me know if you've finished reviewing, so I can merge :))

**Comment by WebSnke** [comment]
> (Btw, let me know if you've finished reviewing, so I can merge :))

It is ready to be merged.

**Comment by WebSnke** [comment]
> > Did you use Goggle Translate to translate the website?
> 
> I did not, we relied on volunteer translations, but then, if it's a language I'm unfamiliar with, I couldn't really tell if they themselves used Google Translator.

I was asking because Google Translate is sometimes a bit weird. DeepL is much better in almost any way.

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**